### ISSUE TITLE
Data sets with calculations

### ISSUE THREAD

**Issue by DaxBourcier** [canadian-federation-of-medical-students/CFMS-Issue-Tracker#67]
Just wondering if we will be able to have the data sets with calculations included by AGM? 

If not perhaps we should remove them from the front end:

![image](https://user-images.githubusercontent.com/62123024/92414967-208aa180-f12d-11ea-8c85-8a876b5e5b9f.png)

Thanks!

**Comment by Jrschellenberg** [comment]
@DaxBourcier can you confirm if all Data ratios (Excluding CaRMS specific) are present here? https://cfms.org/health-human-resources-platform/

**Comment by DaxBourcier** [comment]
I just checked the ratio and the ones that are there now look perfect!

There was the CaRMS one that I didn’t find, which is as follows:
-	Ratio of “CMG applicants to specialty who ranked specialty as 1st choice” divided by “for specialty”
-	We would call it “Ratio of CMG applicants who ranked specialty as 1st choice to Number of CMG seats”
-	Note that this will only work for the Canadian regions because they’re the only ones that have data for the numerator. So for example: Atlantic should show say 0.6 and NS, NB, PEI and NL would show “-”

I know we talked about this one and weren’t sure because we hadn’t confirmed the CaRMS data, but now that we do this ratio will be very relevant to med students. It basically shows the competitiveness of specialties.

Thanks!!

**Comment by Jrschellenberg** [comment]
Hey there @DaxBourcier I just uploaded 2020 CaRMS data as a test to Staging,

https://hhrtf-staging.cfms.org/

can you verify the CaRMS data. Just query for year 2020

**Comment by DaxBourcier** [comment]
Hey Justin,

There's still a few ratios that don't show up on the map when queried:

![image](https://user-images.githubusercontent.com/62123024/93080067-81e9cd80-f663-11ea-8c1a-4c72fc2f5d74.png)

(see the summary too)

Also on this image is all Ont Qu/ and Atlantic should also have the CaRMS ratio but its not showing up.

Thanks!

**Comment by Jrschellenberg** [comment]
For the CaRMS Data. This is working as Expected. There is no Data for General Pathology for Ont and Que.


Can you clarify which Data Ratios aren't showing up and I can check for you!

**Comment by Jrschellenberg** [comment]
@DaxBourcier Let me rephrase what I said before. There is no General Pathology data that CaRMS has provided for the year of 2019.

This is working as expected, the Application just lacks the Data provided by the Government sources.
